utils/apiLog.js

```diff
@@ -68,10 +73,11 @@ function mapApiResultsToLogObject(resultObj, logObject, retval) {
  * @param {Object} resultObj - API result object
  * @param {Object} logObject - from the request object
  * @param {Object or Array} retval - the returned object
+ * @param {Integer} recordCount - a custom recordCount
```

Comment:
pls. rename this to "recordCountOverride" and call it out as optional in the field description in the @param comments